.github/workflows/e2e-tests-main-devnet.yml

```diff
@@ -532,6 +532,148 @@ jobs:
           ALEPH_NODE_BINARY: aleph-release-node/aleph-node
         run: ./.github/scripts/test_multiple_restarts.sh
 
+  check-runtime-change:
+    name: Inspect whether runtime version has been changed (comparing to main)
+    runs-on: ubuntu-latest
+    env:
+      OLD_REPO_PATH: old-repo
+      NEW_REPO_PATH: new-repo
+      OUTPUT_NAME: diff
+    outputs:
+      runtime-updated: ${{ steps.inspect.outputs.diff }}
+    steps:
+      - name: Checkout old source code (main branch)
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.base_ref }}
+          path: ${{ env.OLD_REPO_PATH }}
+
+      - name: Checkout current source code
+        uses: actions/checkout@v3
+        with:
+          path: ${{ env.NEW_REPO_PATH }}
+
+      - name: Inspect versions
+        id: inspect
+        run: |
+          ./${{ env.NEW_REPO_PATH }}/.github/scripts/check_runtime_change.sh \
```

Comment:
I used this command but still as another step - it was much easier for me to preserve this as separate step and trigger further depending on the output (I couldn't make it clean otherwise)